### PR TITLE
debian: install daemons to /usr/libexec/albatross/ to avoid accidental execution

### LINKED ISSUE #91: Avoid daemons to override sockets of already executing daemons
To avoid accidental executions of albatross_console etc. (which will create and claim the socket). Eventually these programs should as well not start up if the socket is already there and opened (but that may lead to rather intricate semantics).

### PR DESCRIPTION
**Pull request by hannesm**
fixes #91

### CODE REVIEW

**Comment by hannesm** [comment]
this removes the `install.sh` from the packging/Linux directory, since its usefulness is questionable - and I prefer to build system packages and maintain paths etc. only once.

File: packaging/debian/create_package.sh [hunk 1]
```diff
@@ -9,16 +9,16 @@ fi
 
 basedir=$(realpath "$(dirname "$0")"/../..)
 bdir=$basedir/_build/install/default/bin
-tmpd=$basedir/_build/stage
+gtmpd=$basedir/_build/stage
```
Comment (reynir) [inline packaging/debian/create_package.sh:12]: Looks like this was a typo not meant to be committed.

**Review by reynir — CHANGES_REQUESTED** [review]
I will fix the typo now. It looks good to me otherwise. Thanks!

**Comment by hannesm** [comment]
thanks, indeed a typo and me not looking through the diff completely.

**Review by reynir — APPROVED** [review]

**Pull request merged**